File: Unit-1/Week3/FunctionsYou13-5/Assignment13-5/src/ArrayAverage.java
import java.util.Scanner;

public class ArrayAverage {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while (t-- > 0) {
            int n = scanner.nextInt();
            int[] arr = new int[n];
            for (int i = 0; i < n; i++) {
                arr[i] = scanner.nextInt();
            }
            System.out.println(average(arr, n));
        }
    }
    public static int average(int[] arr, int n) {
        if (n > 0) {
            int sum = 0;
            for (int i = 0; i < n; i++) {
                sum = sum + arr[i];
            }
            int avg = sum / n;
            return (int) avg;
        } else
            return 0;
    }
}

File: Unit-3/week4/Day1/AssignmentKotlin/src/ArrayAverage.kt
import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    val a = intArrayOf()
    println(avg(a))
}

private fun avg(a: IntArray): Double {
    val n = a.size
    if (n == 0)
        return 0.0
    var sum = 0
    for (i in 0 until n) {
        sum += a[i]
    }
    return (sum / n).toDouble()
}